sympy/solvers/solveset.py

```diff
@@ -1493,7 +1493,7 @@ def add_type(lhs, rhs, x):
       `\_solve\_class`.
       For example: for exponential equations it becomes
       `\_is\_exponential` and `\_solve\_expo`.
-    - The `identifying helpers`should take two input parameters,
+    - The `identifying helpers` should take two input parameters,
```

Comment:
These all should use double backticks, so they don't render as mathematics. It looks like the above uses backslashes to make it render properly as math, but it should be rendered as code.

Reply:
Oh, I see. I have used backslash everywhere in the documentation as an escape sequence. I will change it.